test/client.test.js

```diff
@@ -249,31 +249,21 @@ describe('ClientTester', () => {
       limit: 10,
       offset: 0,
       flags: 0,
-      senderConditions: {
-        conditions: [
-          { value: "*TemperatureSensor*", type: 1 } // Wildcard
-        ]
-      },
+      senderConditions: ["*CDPLoggerDemoApp.Sine*"],
       dataConditions: {
-        pressure: {
-          conditions: [
-            { value: "high", type: 0 } // Exact
```

Comment:
Again, I'm not sure about removing match type option - the wildcard was quite useful and there might be plans in the future to also implement regex, so I would keep the configurable field